codegen/type_converter.go

```diff
@@ -71,14 +71,26 @@ type TypeConverter struct {
 	Helper        PackageNameResolver
 	uninitialized map[string]*fieldStruct
 	fieldCounter  int
+
+	requestTypeHelper ConvertOptions
```

Comment:
fieldName and typeName dont read cleanly.

Reply:
what do you suggest